backend/ops_api/ops/utils/change_requests.py

```diff
@@ -0,0 +1,86 @@
+from datetime import date
+
+from flask import current_app
+
+from models import (
+    AgreementChangeRequest,
+    BudgetLineItemChangeRequest,
+    ChangeRequest,
+    ChangeRequestStatus,
+    Division,
+    Notification,
+)
+
+
+def get_expires_date():
+    return date(2031, 12, 31)  # what should this be?
+
+
+def create_notification_of_new_request_to_reviewer(change_request: ChangeRequest):
+    if not isinstance(change_request, AgreementChangeRequest):
+        return  # we only have messages here for Agreement related change requests for now
+    agreement_id = change_request.agreement_id
+    division_director_ids = set()
+    division: Division = current_app.db_session.get(Division, change_request.managing_division_id)
```

Comment:
`change_request.managing_division_id` can be `NULL` and so `division` would be `None` and line 25 would throw an exception in that case - maybe return early from the function if `division` is `None` here?